qsignature/src/org/qcmg/sig/util/SignatureUtil.java

```diff
@@ -580,65 +569,71 @@ public static void getDataFromBespolkeLayout(File file, int minCoverage, int min
 	}
 	
 	/**
-	 * Convert float array containing allele percentages for ACGT bases into a short.
+	 * Convert float array containing allele percentages for ACGT bases into a byte.
 	 * 
-	 * If over 90% in A, then will return 2000
-	 * If over 90% in C, then will return 200
-	 * If over 90% in G, then will return 20
-	 * If over 90% in T, then will return 2
-	 * If het for AC, will return 1100
-	 * If het for AG, will return 1010
-	 * If het for AT, will return 1001
-	 * If het for CG, will return 0110
-	 * If het for CT, will return 0101
-	 * If het for GT, will return 0011
+	 * If over 90% in A, then will return 1000000 (binary form)
```

Comment:
I think it'd be good to have this mapping between ints and well-defined genotypes explicit somewhere as static finals so there aren't so many magic numbers floating about here and in tests

Reply:
I agree - done